lib/cocoapods-downloader/http.rb

```diff
@@ -100,6 +107,13 @@ def extract_with_type(full_filename, type = :zip)
           tar! 'xfj', unpack_from, '-C', unpack_to
         when :txz
           tar! 'xf', unpack_from, '-C', unpack_to
+        when :dmg
+          plist_s = hdiutil! %(attach -plist -nobrowse #{unpack_from} -mountrandom #{unpack_to})
+          plist = REXML::Document.new plist_s
+          xpath = '//key[.="mount-point"]/following-sibling::string'
+          mount_point = REXML::XPath.first(plist, xpath).text.shellescape
```

Comment:
we shouldn't need to shell escape any arguments, as executables are called with an array of arguments, outside of a shell